### PR TITLE
Update starter template

### PR DESCRIPTION
**Pull request by bisharat-akram**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -4,4 +4,6 @@ module.exports = {
   trailingComma: 'es5',
   printWidth: 100,
   tabWidth: 2,
+  printWidth: 120,
```
Comment (WaqarAhmad707) [inline .prettierrc.js:5]: I think `printWidth` is already written above(line#5), please update it.

File: app/demo/layout.tsx
Comment (WaqarAhmad707) [inline app/demo/layout.tsx:1]: Should we create a simple demo layout component instead of empty file?

File: app/layout.tsx
Comment (WaqarAhmad707) [inline app/layout.tsx:1]: Should we create a simple app layout component instead of empty file?

File: icons/package.json
Comment (WaqarAhmad707) [inline icons/package.json:1]: What is the purpose of package.json in icons folder?

File: redux/example.ts
Comment (WaqarAhmad707) [inline redux/example.ts:1]: We've to add at-least one slice so to understand the redux flow and pattern and also should use the useDispatch and useSelector in our any component for understanding purpose.

File: .prettierrc.js (hunk 1 above)
Comment (bisharat-akram) [reply .prettierrc.js:5]: Yeah, I noticed that and removed the additional one.

File: app/demo/layout.tsx
Comment (bisharat-akram) [reply app/demo/layout.tsx:1]: I think there is a simple layout available. Can you please check by manually opening the file.

File: app/layout.tsx
Comment (bisharat-akram) [reply app/layout.tsx:1]: I think there is a simple layout available. Can you please check by manually opening the file.

File: icons/package.json
Comment (bisharat-akram) [reply icons/package.json:1]: This is another app created in the boilerplate already to convert svg to react icons.

File: redux/example.ts
Comment (bisharat-akram) [reply redux/example.ts:1]: Let me discuss it with you.

**Comment by AdilSarwarNU** [comment]
@bisharat-akram @WaqarAhmad707 shouldn't we add API Configuration too and a dummy file to contain API Routes?

**Comment by bisharat-akram** [comment]
@AdilSarwarNU yes We discussed that and I just forgot about it. I am adding the dummy api file.

**Comment by WaqarAhmad707** [comment]
> @bisharat-akram @WaqarAhmad707 shouldn't we add API Configuration too and a dummy file to contain API Routes?

Yes, I think we should add some sample API.

**Comment by WaqarAhmad707** [comment]
@bisharat-akram I think we're missing one thing
- CustomHooks
We can also create a directory for it.

**Pull request closed without merging**